docker/nginx/web.conf

```diff
@@ -18,18 +18,4 @@ server {
     location / {
         proxy_pass http://web:8000/;
     }
-
-    # Sendfile support to serve non-html files under "/dashboard/<project_slug>/downloads/<type>/<version_slug>"
```

Comment:
This block is completely removed because we are not serving non-html files from web anymore.

Reply:
Let's keep it for now. We will continue to support download links via the dashboard, no? We need to at least not break those URL's.

Reply:
I understand your point. I think this block is useful on production, but I'm not sure if it's worth to keep this code in the docker development configs.

Reply:
I'd like to keep them in sync, but don't feel strongly, so I guess I'm OK dropping this.